packages/serial/src/ZWaveSocket.ts

```diff
@@ -6,6 +6,10 @@ export type ZWaveSocketOptions =
 	| Omit<net.TcpSocketConnectOpts, "onread">
 	| Omit<net.IpcSocketConnectOpts, "onread">;
 
+export interface DisconnectError extends Error {
+	disconnected: boolean;
+}
+
 /** A version of the Z-Wave serial binding that works using a socket (TCP or IPC) */
 export class ZWaveSocket extends ZWaveSerialPortBase {
```

Comment:
That's the wrong file! This is the
> serial binding that works using a socket (TCP or IPC)

You wanted to edit `ZWaveSerialPort.ts`, which uses the serialport binding.

Reply:
I think it could be applied in both but ok I will move it to the other file